src/arrays/Arrays.php

```diff
@@ -1,57 +1,81 @@
 <?php
+
 /**
- * Create the Arrays Class and implement the ArraysInterface and following methods:
+ * Arrays Class implements the ArraysInterface and following methods:
  * repeatArrayValues(), getUniqueValue(), groupByTag()
  * See details below.
  *
- * Note: Use next namespace for Arrays Class - "namespace arrays;" (Like in this Interface)
- * About composer autoloading and namespaces you can read here -
- * https://www.phptutorial.net/php-oop/php-composer-autoload/
+ * Php version 8.1.9
+ * 
+ * @category No_Category
+ * @package  No_Package
+ * @author   Andrey Bakayev <andreybakayev@gmail.com>
+ * @license  https://github.com/AndriiBakayev free
+ * @link     https://github.com/AndriiBakayev
  */
 
 namespace arrays;
 
+/**
+ * Arrays Class implements the ArraysInterface and following methods:
+ * repeatArrayValues(), getUniqueValue(), groupByTag()
+ *  
+ * @category No_Category
+ * @package  No_Package
+ * @author   Andrey Bakayev <andreybakayev@gmail.com>
+ * @license  https://github.com/AndriiBakayev free
+ * @link     https://github.com/AndriiBakayev
+ */
 class Arrays implements ArraysInterface
 {
     /**
      * The $input variable contains an array of digits
-     * Return an array which will contain the same digits but repetitive by its value
+     * Returns an array which will contain the same digits 
+     * but repetitive by its value
      * without changing the order.
      * Example: [1,3,2] => [1,3,3,3,2,2]
      *
-     * Make sure the next PHPDoc instructions will be added:
-     * @param array $input
-     * @return array
+     * @param array $input array of ints
+     * 
+     * @return array $output array where each value repeated by value times
      */
-    public function repeatArrayValues(array $input): array{
-        $output=[];
-        foreach($input as $key=>$val){
-            for ($i=0; $i<$val;$i++)
-                $output[]=$val;
+    public function repeatArrayValues(array $input): array
+    {
+        $output = [];
+        foreach ($input as $key => $val) {
+            for ($i = 0; $i < $val; $i++) {
```

Comment:
please try to use php array function

Reply:
Tried 2 more algorithms and found maybe 
the most array-functioned method